examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb

```diff
@@ -25,7 +25,7 @@
     "\n",
```

Comment:
<p>Let's take the font size down a notch</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>it doesnt seem to render large on gihub, or my laptop. Yet in the git diff its large. weird. </p>